src/apps/opening-hours-editor/helper.ts

```diff
@@ -1,15 +1,25 @@
 import { DplOpeningHoursListGET200Item } from "../../core/dpl-cms/model";
 
-const formatCmsEventsToFullCalendar = (
+function formatDateTimeString(date: string, time: string) {
+  return `${date}T${time}:00`;
+}
+
+export function createCmsEventId(title: string, startDay: Date) {
+  return `${title}-${startDay.toISOString()}`;
+}
```

Comment:
I'm unsure if this is necessary. The way i see it, these ID's are not necessary to differentiate what is being created/edited in the Calender, and i would think that the DB will create necessary ids when the openings hours are created. 

I could be wrong, but i think you should check this.

Reply:
It is necessary to edit events, but the id should come from the API in the future.